<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OSRSmath Optimal Gear Calculator - By Palfore</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="12" column="2">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="xp_rate">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Experience Rate (k/hr):</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="5">
     <widget class="QPlainTextEdit" name="input_text_box"/>
    </item>
    <item row="12" column="0">
     <widget class="QPushButton" name="update_database">
      <property name="text">
       <string>Update Database</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="evaluate">
      <property name="text">
       <string>Evaluate!</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="15">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Alternatives:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QRadioButton" name="player_stats">
        <property name="text">
         <string>Player Stats</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">EditingOptionGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="opponents">
        <property name="text">
         <string>Opponents</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">EditingOptionGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="ignore">
        <property name="text">
         <string>Ignore</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">EditingOptionGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="adjustments">
        <property name="text">
         <string>Adjustments</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">EditingOptionGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="5" rowspan="8">
     <widget class="QTableWidget" name="alternate_gear_bonuses">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Stab</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Slash</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Crush</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Ranged</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Magic</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Melee Str</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Ranged Str</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Magic Str</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Attack Speed</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Cost</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Difference</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="15" column="0" colspan="6">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="progress_task">
        <property name="text">
         <string>Waiting to Evaluate...        </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2" rowspan="10">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QTableWidget" name="best_in_slot_bonuses">
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>Stab</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Slash</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Crush</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Ranged</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Magic</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Melee Str</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Ranged Str</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Magic Str</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Attack Speed</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Cost</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Bonuses</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="1" column="0">
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="2" column="0">
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="3" column="0">
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="4" column="0">
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="5" column="0">
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="6" column="0">
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="7" column="0">
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="8" column="0">
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
        <item row="9" column="0">
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTableWidget" name="best_in_slot">
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>Head</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Cape</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Neck</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Ammo</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Weapon</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Body</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Shield</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Gloves</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Feet</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Ring</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Item</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="4" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="12" column="5">
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="text">
       <string>Search Item on Wiki</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>Helmet</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>Cape</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>Amulet</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_7">
        <property name="text">
         <string>Ammo</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_8">
        <property name="text">
         <string>Weapon</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_10">
        <property name="text">
         <string>Body</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_9">
        <property name="text">
         <string>Shield</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_11">
        <property name="text">
         <string>Gloves</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_12">
        <property name="text">
         <string>Boots</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_13">
        <property name="text">
         <string>Ring</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">AlternateGearSelectionGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="4" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Alternate  Gear Choices</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="11">
     <widget class="Line" name="line_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="training_skill">
      <item>
       <property name="text">
        <string>Train Attack</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Train Strength</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Train Defence</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Train Ranged</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Train Mage</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Optimal Offensive Gear</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="EditingOptionGroup"/>
  <buttongroup name="AlternateGearSelectionGroup"/>
 </buttongroups>
</ui>
